public/data/job-openings-data.json

```diff
@@ -213,6 +213,44 @@
         }
       ]
     },
+    {
```

Comment:
Please update the link only rather update whole object
Suggest to create new branch from devfestahm:develop